classes/controller/AdminController.php

```diff
@@ -4166,8 +4166,21 @@ protected function processBulkStatusSelection($status)
                 /** @var ObjectModel $object */
                 $object = new $this->className((int) $id);
                 $object->active = (int) $status;
-                $result &= $object->update();
+                $isUpdated = (bool) $object->update();
+                $result &= $isUpdated;
+
+                if (!$isUpdated) {
+                    $this->errors[] = $this->trans('Can\'t update #%id% status', ['%id%' => (int) $id], 'Admin.Notifications.Error');
+                }
             }
+
+            if ($result) {
+                $this->redirect_after = self::$currentIndex . '&conf=5&token=' . $this->token;
+            }
+
+            $this->errors[] = $this->trans('An error occurred while updating the status.', [], 'Admin.Notifications.Error');
+        } else {
+            $this->errors[] = $this->trans('You must select at least one element to perform a bulk action.', [], 'Admin.Notifications.Error');
```

Comment:
```suggestion
            $this->errors[] = $this->trans('You need to select at least one item to use the bulk action.', [], 'Admin.Notifications.Error');
```

I changed the wording to try to stay consistent with the existing one. :)

Reply:
Done ! Thanks @Julievrz